src/docs/product/dashboards/custom-dashboards/index.mdx

```diff
@@ -89,15 +89,29 @@ transactions'.
 
 ## Data Set Selection
```

Comment:
I've made the wording edits that will apply on this page, but most of this content is being moved to a new page in [PR 5065](https://github.com/getsentry/sentry-docs/pull/5065).